src/components/marker/timeline-preview.scss

```diff
@@ -8,10 +8,13 @@
   padding-top: 40px;
   z-index: 1;
   &:not(.xs-player) {
-    .header:hover {
+    .header{
       background: rgba(0, 0, 0, 0.6);
       border-radius: $roundness-1;
```

Comment:
you can shorten to

.header {
      background: rgba(0, 0, 0, 0.6);
      border-radius: $roundness-1;

     &:hover {
         background: rgba(0, 0, 0, 0.7);
     }
}